tutorials/plugins/editor/making_plugins.rst

```diff
@@ -448,3 +449,27 @@ Use the following code to register a singleton from an editor plugin:
 
     func _exit_tree():
         remove_autoload_singleton(AUTOLOAD_NAME)
+
+ .. code-tab:: csharp
+
+    #if TOOLS
+    using Godot;
+
+    [Tool]
+    public partial class CustomDock : EditorPlugin
+    {
+        // Replace this value with a PascalCase autoload name, as per the GDScript style guide.
+        const string AUTOLOAD_NAME = "SomeAutoload";
+
+        public override void _EnterTree()
+        {
+            // The autoload can be a scene or script file.
+            AddAutoloadSingleton(AUTOLOAD_NAME, "res://addons/my_addon/some_autoload.tscn");
```

Comment:
@raulsntos I'm happy to address the other examples and make any necessary adjustments. Please provide me with additional details or specific feedback, and I'll incorporate those changes into this PR.

Reply:
This can be done in a follow-up PR, but these are the changes that would need to be made:

- The `CustomNode` example should change the `script` and `texture` paths to use PascalCase.
- The `CustomDock` example should change the `dock` scene path to use PascalCase and add the missing `res://` part.
- The `CustomDock` example should also add the `private` accessibility modifier to the `dock` field and rename it to `_dock`.
- The `Subfolder` in the first example should also use PascalCase.

Reply:
This can be done in a follow-up PR.

- The `CustomNode` example should change the `script` and `texture` paths to use PascalCase.
- The `CustomDock` example should change the `dock` scene path to use PascalCase and add the missing `res://` part.
- The `CustomDock` example should also add the `private` accessibility modifier to the `dock` field and rename it to `_dock`.
- The `Subfolder` in the first example should also use PascalCase.

Reply:
I don't think a new issue is necessary, you can just link to this comment for context.